### PR TITLE
Fix passing flag --discovery-token-unsafe-skip-ca-verification to kubeadm join

### PR DESCRIPTION
**Pull request by shashidharatd**
Address the comment in https://github.com/kubernetes/kubernetes-anywhere/pull/488/files#r153078767

I have tested the changes in my local gce environment and it has passed for `"$KUBEADM_VERSION" == "stable"`

/cc @xiangpengzhao @luxas @pipejakob

### CODE REVIEW

File: phase2/kubeadm/configure-vm-kubeadm-node.sh
Comment (xiangpengzhao) [inline phase2/kubeadm/configure-vm-kubeadm-node.sh]: @shashidharatd can we make sure the "stable" version is always greater than 1.8.0? I guess so as the current version found in the link is v1.8.4
https://github.com/kubernetes/kubernetes-anywhere/blob/7eee9cb5d0767619f1833fb5242d2543b04844dd/phase2/kubeadm/do#L39

dl.k8s.io/release/stable.txt

**Review by xiangpengzhao — COMMENTED** [review]
@shashidharatd this is so cool! Thanks! Defer to @luxas for another review.

File: phase2/kubeadm/configure-vm-kubeadm-node.sh
Comment (shashidharatd) [reply phase2/kubeadm/configure-vm-kubeadm-node.sh]: @xiangpengzhao, as you pointed out, current stable version is v1.8.4 and it will always go up. while pushing the release to GCS there is check which ensures pointers like stable.txt and latest.txt are updated only if the newer version is greater than the existing version.

**Review by luxas — APPROVED** [review]
/lgtm

Thanks @shashidharatd!

**Comment by luxas** [comment]
I now got merge rights here (woot!) so merging

**Comment by xiangpengzhao** [comment]
https://github.com/kubernetes/kubernetes/issues/56091 is green. Thanks @shashidharatd ! :+1:

**Pull request merged**
